### PR TITLE
Add a convenient way to get USER_AGENT values and add new fake user_agent values, see issue #336

### PR DESCRIPTION
**Pull request by haruntuncay**
**Added a way to access user_agent values using method calls through `faker.internet()` instead of resolving them through `faker.resolve(...)`.**
Example:
`faker.internet().userAgent(USER_AGENT.CHROME); //returns a CHROME user_agent value.`

**Also, added a way to get a random user_agent value without pre-selecting of browser vendor.**
Example:
`faker.internet().userAgentAny(); //returns a random user_agent value.`

**Increased the number of user_agent values in `internet.yml` file to 10 value for each browser**.

### CODE REVIEW

File: src/main/java/com/github/javafaker/Internet.java [hunk 1]
```diff
@@ -297,4 +297,47 @@ public String uuid() {
...
+
+        String userAgentKey = "internet.user_agent." + agent.toString();
+        return faker.fakeValuesService().resolve(userAgentKey, this, faker);
+    }
+
+    public String userAgentAny(){
+        return userAgent(null);
+    }
+
+    public enum USER_AGENT {
```
Comment (codingricky) [inline src/main/java/com/github/javafaker/Internet.java]: Could you use standard java conventions for naming so CamelCase here? Thanks

File: src/test/java/com/github/javafaker/InternetTest.java [hunk 2]
```diff
@@ -276,4 +277,16 @@ public void testFarsiIDNs() {
         assertThat(f.internet().safeEmailAddress(), not(isEmptyOrNullString()));
         assertThat(f.internet().url(), not(isEmptyOrNullString()));
     }
+
+    @Test
+    public void testUserAgent() {
+        Internet.USER_AGENT[] agents = Internet.USER_AGENT.values();
+        for(Internet.USER_AGENT agent : agents){
+            assertThat(faker.internet().userAgent(agent), IsNot.not(isEmptyOrNullString()));
```
Comment (codingricky) [inline src/test/java/com/github/javafaker/InternetTest.java]: There's the static import already for IsNot so you shouldn't need to fully qualify it

File: src/test/java/com/github/javafaker/InternetTest.java [hunk 3]
```diff
@@ -276,4 +277,16 @@ public void testFarsiIDNs() {
...
+            assertThat(faker.resolve("internet.user_agent." + agent.toString()), IsNot.not(isEmptyOrNullString()));
```
Comment (codingricky) [inline src/test/java/com/github/javafaker/InternetTest.java]: I don't think you need this assert here. You're testing that the resolve method works and that the yml is correctly populated. I feel like your assert above should cover that.

File: src/main/java/com/github/javafaker/Internet.java [hunk 4]
```diff
@@ -297,4 +297,47 @@ public String uuid() {
     private <T> T random(T[] src) {
         return src[faker.random().nextInt(src.length)];
     }
+
+    public String userAgent(USER_AGENT userAgent){
```
Comment (codingricky) [inline src/main/java/com/github/javafaker/Internet.java]: could you add a space between the closing brace and the opening parenthesis?

```
userAgent) {
```
add apply that to all the added methods.. sorry I really should add some linting so it's easier for contributors

**Comment by codingricky** [comment]
thanks so much for the PR, I've added a few small comments

**Comment by haruntuncay** [comment]
Thank you for your review. I will deal with the issues and update my PR.

File: src/test/java/com/github/javafaker/InternetTest.java (hunk 3 above)
Comment (haruntuncay) [reply src/test/java/com/github/javafaker/InternetTest.java]: So you mean the second test is unnecessary ?

**Comment by haruntuncay** [comment]
Hi @codingricky, I have fixed the issues you've pointed out. Thank you for the review.

**Pull request merged**